packages/web/src/generic.ts

```diff
@@ -34,3 +34,29 @@ export const inject = (
 
   document.head.appendChild(script);
 };
+
+export const track = (
+  event: string,
+  properties?: Record<string, AllowedPropertyValues>,
+): void => {
+  if (!properties) {
+    window.va?.('track', { name: event });
+    return;
+  }
+
+  try {
+    const props = parseProperties(properties, {
+      strip: !isDevelopment(),
+    });
+
+    window.va?.('track', {
+      name: event,
+      data: props,
+    });
+  } catch (err) {
+    if (err instanceof Error && isDevelopment()) {
+      // eslint-disable-next-line no-console
+      console.error(err);
+    }
+  }
```

Comment:
Let's use name here as it's already confusing what's an event what's the name etc.

```suggestion
  name: string,
  properties?: Record<string, AllowedPropertyValues>,
): void => {
  if (!properties) {
    window.va?.('track', { name });
    return;
  }

  try {
    const props = parseProperties(properties, {
      strip: !isDevelopment(),
    });

    window.va?.('track', {
      name,
      data: props,
    });
  } catch (err) {
    if (err instanceof Error && isDevelopment()) {
      // eslint-disable-next-line no-console
      console.error(err);
    }
  }
```